ZUT-PokerPlanningClient/src/app/project-planning/game/user-card-item/user-card-item.component.html

```diff
@@ -0,0 +1,12 @@
+<div class='game-user-element'>
+    <div class='test'>
+        <div class='user-card'>
+            <div class='initials'>
+                {{userItem.name[0]}}{{userItem.surname[0]}}
```

Comment:
I'm curious what would happen if undefined/null will be returned from the server and how we would like to display it then